crud/src/main/java/com/redhat/lightblue/rest/crud/cmd/BulkRequestCommand.java

```diff
@@ -50,6 +53,7 @@ public CallStatus run() {
             try {
                 req = getJsonTranslator().parse(BulkRequest.class, JsonUtils.json(request));
             } catch (Exception e) {
+                metricCtx.markRequestException(e, e.getMessage());
                 LOGGER.error("bulk:parse failure: {}", e);
                 return new CallStatus(Error.get(RestCrudConstants.ERR_REST_ERROR, "Error parsing request"));
```

Comment:
Again for these I would use the `Error` type returned from `Error.get(RestCrudConstants.ERR_REST_ERROR, "Error parsing request")`.

Reply:
Fixed.